recipe/meta.yaml

```
@@ -1,7 +1,11 @@
 {% set version = "5.0.0" %}
 {% set sha256 = "019f23c2192df793ac746595e94a403908749f8e0c484b403476d2611dd20970" %}
 
-{% set build_number = "1005" %}
+{% set build_number = "1006" %}
```

Is there any need to bump the build number? I guess there is no need to redeploy the other variants.
```suggestion
{% set build_number = "1005" %}
```

---

I guess that this is fine to leave it at 1005.